ext_mongodb.php

```diff
@@ -620,6 +620,78 @@ class BulkWriteException extends WriteException {}
 /* {{{ BSON and Serialization Classes */
 namespace MongoDB\BSON;
 
+/* {{{ Interfaces */
+interface TypeWrapper
+{
+	static public function createFromBSONType(\MongoDB\BSON\Type $type) : \MongoDB\BSON\TypeWrapper;
+	public function toBSONType() : \MongoDB\BSON\Type;
+}
+
+interface BinaryInterface
+{
+	public function __construct(string $data, int $type);
+	public function getType() : int;
+	public function getData() : string;
+	public function __debugInfo() : array;
```

Comment:
Why is there no `__toString()` here? [HHVM-232](https://jira.mongodb.org/browse/HHVM-232) is resolved.